### PR TITLE
Add cors to the sample app

### PR DESCRIPTION
**Pull request by blittle**
This allows us to use the sample app bff to get data in other contexts

### CODE REVIEW

**Comment by gurpreetsainisalesforce** [comment]
> This allows us to use the sample app bff to get data in other contexts

What would be an example of "other" contexts?

File: packages/@sfcc-core/core-graphql/src/graphql.ts [hunk 1]
```diff
@@ -139,6 +139,14 @@ export class CoreGraphQL {
             this.apolloServer.applyMiddleware({
                 app: expressApp,
                 path: apiPath,
+                cors: apiConfig.COMMERCE_CORS
```
Comment (blittle) [inline packages/@sfcc-core/core-graphql/src/graphql.ts:142]: By default, right now CORS is enabled. Turn it off unless an environment variable is set.

File: packages/storefront-lwc/scripts/runtime.js [hunk 2]
```diff
@@ -105,6 +105,7 @@ function validateConfig(config) {
         sess.cookie.secure = true; // serve secure cookies
     }
 
+    sampleApp.expressApplication.disable('x-powered-by');
```
Comment (blittle) [inline packages/storefront-lwc/scripts/runtime.js:108]: This is unrelated, but it is generally security best practice to not tell the client what is powering the API.

**Comment by blittle** [comment]
> > This allows us to use the sample app bff to get data in other contexts
> 
> What would be an example of "other" contexts?

@gurpreetsainisalesforce for example, we have a component that needs to run in communities.

File: packages/@sfcc-core/core-graphql/src/graphql.ts (hunk 1 above)
Comment (gurpreetsainisalesforce) [reply packages/@sfcc-core/core-graphql/src/graphql.ts:142]: Can we please have some documentation around it?

**Pull request merged**